python/paddle/nn/quant/quant_layers.py

```diff
@@ -481,6 +482,112 @@ def forward(self, input):
             data_format=self._data_format)
 
 
+class QuantizedConv2DTranspose(layers.Layer):
+    """
+    The computational logic of QuantizedConv2DTranspose is the same with Conv2DTranspose.
+    The only difference is that its inputs are all fake quantized.
+    
+    Examples:
+       .. code-block:: python
+          import paddle
```

Comment:
add a blank line here for preview, otherwise there will be formatting errors

Reply:
QuantizedConv2DTranspose will not be shown to users. The "ImperativeQuantAware.quantize()" API will replace Conv2DTranspose with QuantizedConv2DTranspose automatically.